src/Files.App/UserControls/StatusBar.xaml

```diff
@@ -193,6 +195,7 @@
 							ToolTipService.ToolTip="{x:Bind Commands.OpenInVSCode.LabelWithHotKey, Mode=OneWay}" />
 						<MenuFlyoutItem
 							x:Name="OpenRepoInVSCodeButton"
+							x:Load="{x:Bind DirectoryPropertiesViewModel.GitBranchDisplayName, Mode=OneWay, Converter={StaticResource NullToFalseConverter}, FallbackValue=False}"
```

Comment:
Nested `x:Load` may cause crashes, so it is better to use `Visiblity` instead.
https://github.com/files-community/Files/pull/12726#discussion_r1241172473

Reply:
Fixed